src/lib/const.ts

```diff
@@ -687,6 +689,15 @@ func spell():
 
 const QUICKSTART_README = `# Fluence Quickstart Template
 
+## Content
+
+- Default Marine service - \`${join(SRC_DIR_NAME, SERVICES_DIR_NAME)}\`.
+- Basic aqua functions - \`${join(SRC_DIR_NAME, AQUA_DIR_NAME)}\`.
+- Fluence HTTP Gateway for proxying Aqua execution - \`${join(
+  SRC_DIR_NAME,
+  GATEWAY_DIR_NAME,
```

Comment:
Instead of joining paths yourself, the easier to maintain option would be to use getServicesDir function and others from paths.ts. Also if this part of readme is supposed to be completely the same as the one bellow I would just reuse it instead of duplicating it

Reply:
I cannot use functions like `getServicesDir` because they reference `projectRootDir` and it cause runtime error.
Agree with point about duplication though

Reply:
maybe something is wrong with the order of operations. projectRootDir is set when initing the project

Reply:
You should be able to use these functions but make sure you use them after projectRootDir is set

Reply:
How exactly can i make sure that?

Reply:
you would have to look at the code and see the order of stuff happening

Reply:
and in the template you would use `relative(pathToReadme, pathToTheActualThings)`

Reply:
It works if i put `getServicesDir` inside function. But when i use it in constant in top-level scope, it fails. I don't want to convert constants to functions for the sake of that constant

Reply:
Seems like circular dependency between files because some constants aren't initialized